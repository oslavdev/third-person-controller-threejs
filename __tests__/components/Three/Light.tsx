import { render } from "@testing-library/react";
import Lights from "@/components/Three/Light";

describe('Light', () => {

  const LightComponent = render(<Lights/>);

  it('Snapshot', () => {
   
    expect(
      LightComponent
    ).toMatchSnapshot()
  });

})  
