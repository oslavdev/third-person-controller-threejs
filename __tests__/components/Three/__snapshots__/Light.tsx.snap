// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Light Snapshot 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <ambientlight
        intensity="0.1"
      />
      <directionallight
        intensity="0.2"
        position="40,10,5"
      />
      <directionallight
        intensity="1.3"
        position="10,420,100"
        shadow-camera-bottom="-10"
        shadow-camera-far="10"
        shadow-camera-left="-30"
        shadow-camera-right="10"
        shadow-camera-top="40"
        shadow-mapsize-height="1024"
        shadow-mapsize-width="1024"
      />
      <spotlight
        intensity="0.5"
        position="90,100,50"
      />
    </div>
  </body>,
  "container": <div>
    <ambientlight
      intensity="0.1"
    />
    <directionallight
      intensity="0.2"
      position="40,10,5"
    />
    <directionallight
      intensity="1.3"
      position="10,420,100"
      shadow-camera-bottom="-10"
      shadow-camera-far="10"
      shadow-camera-left="-30"
      shadow-camera-right="10"
      shadow-camera-top="40"
      shadow-mapsize-height="1024"
      shadow-mapsize-width="1024"
    />
    <spotlight
      intensity="0.5"
      position="90,100,50"
    />
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
